homework37/src/main/java/com/vadim/tkach/lesson37/NumberUtils.java

```diff
@@ -0,0 +1,24 @@
+package com.vadim.tkach.lesson37;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NumberUtils {
+    public static int getsum(List<Number> numbers) {
+        int sum = 0;
+        for (Number number : numbers) {
+                sum += number.intValue();
+        }
+        return sum;
+    }
```

Comment:
Ваш метод зможе нормально опрацювати тільки масив інтів. Краще використати `doubleValue()` щоб метод зміг корректно опрацювати `int` / `long` / `float` / `double` типи.

```suggestion
    public static double getSum(List<Number> numbers) {
        double sum = 0;
        for (Number num : numbers) {
            sum += num.doubleValue();
        }
        return sum;
    }
```